vector-app/src/androidTest/java/im/vector/app/ui/robot/OnboardingRobot.kt

```diff
@@ -32,12 +32,17 @@ import im.vector.app.R
 import im.vector.app.core.resources.BooleanProvider
 import im.vector.app.espresso.tools.waitUntilViewVisible
 import im.vector.app.features.DefaultVectorFeatures
+import im.vector.app.features.debug.features.DebugFeatureKeys
+import im.vector.app.features.debug.features.DebugVectorFeatures
 import im.vector.app.waitForView
 
 class OnboardingRobot {
 
+    // Tchap: Use different onboarding variant to run the tests
```

Comment:
we don't change anymore the onboarding variant, we only enable the keyback up for the tests here

```suggestion
    // Tchap: Override default feature values for tests
```